protocol/app/app.go

```diff
@@ -1156,6 +1156,10 @@ func New(
 		[]string{
 			lib.GovModuleAddress.String(),
 		},
+		app.PricesKeeper,
+		app.ClobKeeper,
+		&app.MarketMapKeeper,
```

Comment:
Why does this need to be passed in differently?

Reply:
Its because the keeper methods in marketmapper use a different convention and take a pointer receiver.

`func (k *Keeper) GetMarket(ctx sdk.Context, tickerStr string) (types.Market, error) {
	return k.markets.Get(ctx, types.TickerString(tickerStr))
}`